src/librustc/traits/select.rs

```diff
@@ -808,26 +825,24 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
 
             ty::Predicate::ConstEvaluatable(def_id, substs) => {
                 let tcx = self.tcx();
-                match tcx.lift_to_global(&(obligation.param_env, substs)) {
-                    Some((param_env, substs)) => {
```

Comment:
I'd prefer if one of two things happened here (and in other places `match`-ing over `lift_to_global` results):
* land a PR on master replacing the `match`es with `if let`s (you can just r=me that if you want)
* keep the `match` and either use `Option::filter` or just `match` on the `bool` predicate

That is, I would prefer if indentation didn't change (and it only does because of the `match` -> `if` change).

Reply:
Why do you not want indention to change? I like less rightward drift, and I definitely don't like matching on bools.

Reply:
It's fine to change, I just want to be able to read the diff without mentally pretending nothing changed without checking, where indentation is involved.

Reply:
I changed the indentation to reduce the diffs in this PR.

Reply:
@eddyb you can append `?w=1` to the diff url to have whitespace insensitive diffs https://github.com/rust-lang/rust/pull/61871/files?w=1

Reply:
Nice. I also reordered some branches which that can't deal with though =P